db/migrations/20221026215921_add-breaches-table.js

```diff
@@ -0,0 +1,27 @@
+exports.up = knex => {
+  return knex.schema
+    .createTable('breaches', table => {
+      table.increments('id').primary()
+      table.string('name').default('').unique()
+      table.string('title').default('')
+      table.string('domain').default('')
+      table.date('breach_date').notNullable()
+      table.timestamp('added_date')
+      table.timestamp('modified_date').defaultTo(knex.fn.now())
+      table.integer('pwn_count').defaultTo(0)
+      table.text('description').default('')
+      table.string('logo_path').default('')
+      table.specificType('data_classes', 'character varying(255)[]')
+      table.boolean('is_verified').defaultTo(false)
+      table.boolean('is_fabricated').defaultTo(false)
+      table.boolean('is_sensitive').defaultTo(false)
+      table.boolean('is_retired').defaultTo(false)
+      table.boolean('is_spam_list').defaultTo(false)
+      table.boolean('is_malware').defaultTo(false)
```

Comment:
The booleans? They currently are relevant for because we use them in a few calls. Some examples: 
* to filter breaches when scanning emails
* to get breaches for a user's verified / unverified emails, we also filter based on these booleans